# Request
string filename
float64 duration
---
# Response
bool success
